src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.HandleDictionary.cs

```diff
@@ -40,34 +40,27 @@ public static partial class JsonSerializer
                 // Handle DataExtension.
                 if (ReferenceEquals(jsonPropertyInfo, state.Current.JsonClassInfo.DataExtensionProperty))
```

Comment:
I think we just need to remove lines 41-45 (and the WriteExtensionData method) + update tests.

Reply:
I opened another PR https://github.com/dotnet/corefx/pull/38306